README.md

```diff
@@ -1,17 +1,86 @@
-## Documentation Guidelines
+# Documentation Guidelines
 
-### 🚀 Contributing
+## 🚀 Contributing
 
-### Before Proposing Additions
+## Before Proposing Additions
 
-### Style Guidelines
+## Style Guidelines
 
-#### Code Examples
+### Code Examples
 
-#### Notices
+### Notices
 
-#### Screenshots
+### Screenshots
 
-#### Terminology
+### Terminology
+Appwrite has many services and features with many ways to express the same concepts. To avoid confusion, this section suggests a standard set of terms used across Appwrite documentation to describe features and concepts.
+#### General
+| **Term**      | **Suggested Usage**                                                                                                                                                                                              |
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Appwrite      | Written as "Appwrite" rather than "AppWrite" or "appwrite".                      |
+| Projects      | Each Appwrite instance can have many Appwrite Projects. Use "Appwrite Projects" as a proper noun when referring to the concept, use project or projects when referring to specific projects.                      |
+| Web Console       | Refers to the the web GUI. Can also be referred to as the "Appwrite Web Console" to distinguish from a Mac, Linux, or Windows machine's console or Chrome's developer console.                                                                          |
```

Comment:
It makes more sense to use simpler terms and call it the `Appwrite Dashboard`. As you have rightly pointed out, using `console` makes us need to disambiguate between the OS specific consoles / chrome developer console.